tests/setup.py

```diff
@@ -0,0 +1,13 @@
+from flumes.config import Config
+from flumes.discoverer import Discoverer, DiscovererOptions
+
+#################################
+####### Discoverer Setup ########
+#################################
+
+
+def discoverer_setup():
```

Comment:
```suggestion
@pytest.fixture
def discoverer_setup():
```
Here you should define this function as a fixture so that you can later pass the return value directly as an input parameter to your tests.
This [link](https://www.tutorialspoint.com/pytest/pytest_conftest_py.htm) describes the approach briefly.